app/models/manageiq/providers/azure/cloud_manager.rb

```diff
@@ -142,7 +142,9 @@ def vm_create_evm_managed_snapshot(vm)
       snap_svc.get(ssa_snap_name, resource_group)
     rescue ::Azure::Armrest::NotFoundException, ::Azure::Armrest::ResourceNotFoundException => err
       begin
-        snap_svc.create(ssa_snap_name, resource_group, snap_options)
+        response = snap_svc.create(ssa_snap_name, resource_group, snap_options)
+        # wait a minute at a time, allowing the Job Timeout to handle long-running snapshots here
+        next until snap_svc.wait(response.response_headers) =~ /^succe/i
```

Comment:
This probably can be merged for now, so we can get the fix out. But this condition is too specific. If it doesn't complete successfully, we'll have to wait for the job to time out. The test should be more along the lines of:
```Ruby
next if snap_svc.wait(response.response_headers) =~ /^In progress/i
```
Not sure of the specific "in progress" state.

Same comment for the similar check below.

Reply:
So I discussed that with @djberg96 on last Wednesday and he was of the opinion that there would be an Exception raised in the case of an unsuccessful completion and therefore we wouldn't hit the test regardless.